src/lib/viewers/controls/annotations/DrawingControls.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import ColorPickerControl from '../color-picker/ColorPickerControl';
```

Comment:
Can just be `../color-picker` here to leverage the index file.

Reply:
Good callout, will fix.